README.md

```diff
@@ -34,7 +35,8 @@ contributors for Windows. Mio is working on windows support, once they have it,
 we will push for Windows too.
 
 You will need a nightly rust to build the current code. I suggest using
-[multirust](https://github.com/brson/multirust) to handle different versions.
+[rustup](https://github.com/rust-lang-nursery/rustup.rs) to handle different
```

Comment:
refer to http://rustup.rs instead? That is what the rustup README.md refers to too. One less indirection.